PubSub/src/Connection/Rest.php

```diff
@@ -1,349 +0,0 @@
-<?php
-/**
- * Copyright 2017 Google Inc. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-namespace Google\Cloud\PubSub\Connection;
-
-use Google\Cloud\Core\EmulatorTrait;
-use Google\Cloud\Core\RequestBuilder;
-use Google\Cloud\Core\RequestWrapper;
-use Google\Cloud\Core\RestTrait;
-use Google\Cloud\Core\UriTrait;
-use Google\Cloud\PubSub\PubSubClient;
-
-/**
- * Implementation of the
- * [Google Pub/Sub REST API](https://cloud.google.com/pubsub/docs/reference/rest/).
- *
- * The `PUBSUB_EMULATOR_HOST` environment variable from the gcloud SDK is
- * honored, otherwise the actual API endpoint will be used.
- */
-class Rest implements ConnectionInterface
```

Comment:
I think it'd be better to just mark the classes as deprecated - there's no reason to remove them IMHO